resources/codegen_inputs/colr.rs

```diff
@@ -6,6 +6,7 @@
 table Colr {
     /// Table version number - set to 0 or 1.
     #[version]
+    #[default(0)]
```

Comment:
I think the best approach for this is going to be `#[compile(self.compute_version())]`, and then add a method to `Color` that picks the correct version based on what fields are used.

Reply:
That works, will so change.